packages/create-miniapp-cli/template/server/bin/generate-condo-schema.js

```
@@ -0,0 +1,475 @@
+const fs = require('fs')
+
+const { GraphQLFileLoader } = require('@graphql-tools/graphql-file-loader')
+const { loadSchema } = require('@graphql-tools/load')
+const { introspectionFromSchema, buildClientSchema, printSchema } = require('graphql')
+const get = require('lodash/get')
+const pluralize =  require('pluralize')
+
+
+const { CONDO_CONFIG } = require('~/domains/condo/gql')
```

ESLint

[error] '~/domains/condo/gql' module is restricted from being used by a pattern.

(no-restricted-modules)